ru/turing/courses/lesson3/Shishlova/Registry.java

```diff
@@ -0,0 +1,87 @@
+package ru.turing.courses.lesson3.Shishlova;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Registry<T> {
```

Comment:
Ты по сути делаешь реджистри для `NamedObject` или его потомков - поэтому напиши тут
```suggestion
public class Registry<T extends NamedObject> {
```